packages/block-editor/src/components/block-mover/index.native.js

```diff
@@ -31,37 +68,43 @@ const BlockMover = ( {
 				title={
 					! isFirst
 						? sprintf(
-								/* translators: accessibility text. %1: current block position (number). %2: next block position (number) */
-								__( 'Move block up from row %1$s to row %2$s' ),
+								/* translators: accessibility text. %1: block moving direction (string). %2: location of block - row or order number (string). %3: current block position (number). %4: next block position (number) */
+								__(
+									'Move block %2$s from %1$s %3$s to %1$s %4$s'
```

Comment:
Could you drop a sample end result of this text? not sure I am able to evaluate correctly in my head.

Reply:
It was logic pulled from Lukes Button PR - already spoke with him and it should evaluate with below rules:

%1 is for block `location` e.g.:
* horizontal - position
* vertical - row

%2 firstButtonDirection is for moving `direction`:
* horizontal - left
* vertical - up

So finally for 'first arrow` it will evaluate to:
- vertical position:  Move block up from row `x` to row `y`
- horizontal position: Move block left from position `x` to position `y`

Being honest I didn't test that with a11y on device but I will today and confirm i that works as above.

Reply:
Have checked above and on Android Emulator with TalkBack enable it works as explained above

Reply:
Translations won't work this way. Both technically, both for the person who translates it. We need to explicitly create all variations like this:

__( 'Move block up from row %1 to row %2' )
__( 'Move block down from row %1 to row %2')

__( 'Move block left from position %1 to position %2')
__( 'Move block right from position %1 to position %2')

Reply:
@etoledom Any opinions about the horizontal version of this sentence? 

> Move block left from position %1 to position %2

Just to give you some context the toolbar looks like this in the horizontal alignment:

![Screenshot_20200402_152248_org wordpress android](https://user-images.githubusercontent.com/5032900/78249000-49f82f00-74f6-11ea-92d2-50e7daa17696.jpg)

Columns and Buttons blocks will be this way.

Reply:
Looks good to me, and I definitely agree with `We need to explicitly create all variations`.
How would this look with RTL layout?

Reply:
Sure 👍 @etoledom

Reply:
I have extract the logic to return proper title to mover button. Can you take a look and give me feedback.